test/e2e/daemon_restart.go

```diff
@@ -203,8 +203,9 @@ var _ = Describe("DaemonRestart", func() {
 
 	AfterEach(func() {
 		close(stopCh)
-		expectNoError(DeleteRC(framework.Client, ns, rcName))
+		err := DeleteRC(framework.Client, ns, rcName)
 		framework.afterEach()
```

Comment:
also, I think using defer framework.afterEach() as the first thing called would achieve the desired affect without moving the err from its source.